src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String content = readFile(fromFileName);
+        writeFile(toFileName, content);
+    }
+
+    private String readFile(String fileName) {
+        int supplyAmount = 0;
+        int buyAmount = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] info = line.split(",");
+                if (info[OPERATION_TYPE].equals(BUY)) {
+                    buyAmount += Integer.parseInt(info[AMOUNT]);
+                } else if (info[OPERATION_TYPE].equals(SUPPLY)) {
+                    supplyAmount += Integer.parseInt(info[AMOUNT]);
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("File not found " + e);
```

Comment:
You are throwing a new RuntimeException here, but the message is not very informative. It would be better to include more details about the error. For example, you could include the name of the file that could not be read. Also, you should pass the original exception `e` as a cause to the new RuntimeException to preserve the stack trace.